internal/services/keyvault/key_vault_key_resource.go

```diff
@@ -139,6 +141,62 @@ func resourceKeyVaultKey() *pluginsdk.Resource {
 				ValidateFunc: validation.IsRFC3339Time,
 			},
 
+			"rotation_policy": {
+				Type:     pluginsdk.TypeList,
+				Optional: true,
+				Computed: true,
+				MaxItems: 1,
+				Elem: &pluginsdk.Resource{
+					Schema: map[string]*pluginsdk.Schema{
+						// at least 28 days
+						"expiry_time": {
```

Comment:
this is a duration?
```suggestion
						"expire_after": {
```
or
```suggestion
						"expire_after_time": {
```

Reply:
expiry time implies a specific date time,

Reply:
Going for `expire_after`